lib/pool.js

```diff
@@ -99,29 +105,22 @@ class Pool extends EventEmitter {
     }
   }
 
+  // TODO: Make private?
   get url () {
     return this[kUrl]
   }
 
   get connected () {
-    return this[kConnected]
+    let ret = 0
+    for (const { connected } of this[kClients]) {
+      ret += connected
+    }
+    return ret
```

Comment:
this could be simplified to `this[kClients].some(({ connected }) => !!connected)`

Reply:
connected is an integer

Reply:
Oh yes; sorry I keep thinking this property is a boolean thing when its a number _of_ connected clients.